index.html

```diff
@@ -337,6 +261,35 @@ <h3 class="my_name">Get Started</h2>
         </div>
       </section>
       </footer>
+       <div id="work" class="modal-pop hidden">
+        <div class="modal-contain">
+          <div class="close-btn" onclick="toggle()"><p>&times;</p></div>
+          <h2 id="work_name">Project name goes here</h2>
+          <ul class="lang_list">
+            <li><a href="#">HTML/CSS</a></li> 
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">JavaScript</a></li>
+          </ul> 
+          <img class="work-img" src="img/popimg.png">
+          <div class="mod-desc">
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Ad ipsam, quisquam perferendis dolor labore quas culpa voluptatibus officia, soluta eveniet reiciendis.
+            Ullam a quis tempora provident voluptatem explicabo vero non.<br> <br>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+             Recusandae, debitis? Vitae, quaerat odit animi, ex corporis, in iure est placeat enim facere ratione molestias! Doloribus sit 
+             obcaecati reiciendis. Sapiente, laboriosam.</p>
+             <div class="see-btns">
+               <button type="button" id="see-button-live" class="see-btn live"> See live
+                 <i class="fas fa-external-link-alt"></i>  </button>
+              <button type="button" id="card-button-source"class="see-btn source"> See source
+                  <i class="fab fa-github"></i>    </button>
+             </div>
+            </div>
+             <div class="arrows">
+              <p><i class="fa fa-arrow-left"></i> Previous Project</p>
+              <p>Next Project<i class="fa fa-arrow-right"></i></p>
+             </div>
+        </div>
+      </div>
```

Comment:
- [ ] The effect of modal popup is also required to come from your javascript file, not from your index.html file. Modal popup data is populated with the data from the projects array. Kindly remove all of these and make it dynamically rendered from your js file.